core/os/device/device.proto

```diff
@@ -173,6 +173,8 @@ message Configuration {
   Drivers drivers = 4;
   // Perfetto capability.
   PerfettoCapability perfetto_capability = 5;
+  // Angle package.
+  string angle_package = 6;
```

Comment:
Added this member to the proto buffer, but still not clear to me how it gets filled in or what I need to use it for.